lib/ukdebug/include/uk/debug/crash.h

```diff
@@ -0,0 +1,90 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright (c) 2021, Karlsruhe Institute of Technology. All rights reserved.
+ * Copyright (c) 2023, Unikraft GmbH and The Unikraft Authors.
+ * Licensed under the BSD-3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ */
+
+#ifndef __UK_CRASH_H__
+#define __UK_CRASH_H__
+
+#include <uk/plat/bootstrap.h>
+#include <uk/arch/lcpu.h>
+#include <uk/essentials.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+enum uk_crash_reason {
+	UK_CRASH_REASON_INVALID,
```

Comment:
Since this is in `uk/debug/`, wondering if all of these should have been namespaced as `UK_DEBUG_CRASH` or maybe the header should have instead been `uk/crash.h` like is the case for `uk/trace.h` and `uk/assert.h`

Reply:
So this generally against our conventions, but I agree it makes sense to make an exception here give the ubiquitous nature of `libukdebug`. Will do 👍🏼